app/components/CapacityBar.tsx

```diff
@@ -6,30 +6,30 @@
  * Copyright Oxide Computer Company
  */
 
-import { splitDecimal } from '~/util/math'
+import { BigNum } from '~/ui/lib/BigNum'
+import { percentage, splitDecimal } from '~/util/math'
 
-export const CapacityBar = ({
+export const CapacityBar = <T extends number | bigint>({
```

Comment:
generic is so we can let provisioned and allocated be `number | bigint` while enforcing that they are the same type